### PR TITLE
Show more detailed container statuses

### PR DESCRIPTION
**Pull request by joaquimrocha**
- frontend: Add a container status label to the Pod details view
- frontend: Make the Pod status better show its real status
- frontend: Show more details about the container status in Pod details

### CODE REVIEW

File: frontend/src/components/common/Resource/Resource.tsx [hunk 1]
```diff
@@ -440,7 +444,7 @@ function LivenessProbes(props: { liveness: KubeContainer['livenessProbe'] }) {
   return (
     <Box display="flex" flexDirection="column">
       <LivenessProbeItem>
-        {`http-get, path: ${liveness?.httpGet?.path}, port: ${liveness?.httpGet?.port}, 
+        {`http-get, path: ${liveness?.httpGet?.path}, port: ${liveness?.httpGet?.port},
```
Comment (ashu8912) [inline frontend/src/components/common/Resource/Resource.tsx:448]: do we need this change here??

File: frontend/src/components/common/Resource/Resource.tsx [hunk 2]
```diff
@@ -37,9 +35,11 @@ import Loader from '../../common/Loader';
 import { SectionBox } from '../../common/SectionBox';
 import SectionHeader, { HeaderStyleProps } from '../../common/SectionHeader';
 import SimpleTable, { NameValueTable, NameValueTableRow } from '../../common/SimpleTable';
+import { LightTooltip } from '..';
 import Empty from '../EmptyContent';
 import { DateLabel, HoverInfoLabel, StatusLabel, StatusLabelProps } from '../Label';
 import Link, { LinkProps } from '../Link';
+import { useMetadataDisplayStyles } from '.';
```
Comment (ashu8912) [inline frontend/src/components/common/Resource/Resource.tsx:42]: why this change??

File: frontend/src/components/common/Resource/Resource.tsx [hunk 3]
```diff
@@ -37,9 +35,11 @@ import Loader from '../../common/Loader';
 import { SectionBox } from '../../common/SectionBox';
 import SectionHeader, { HeaderStyleProps } from '../../common/SectionHeader';
 import SimpleTable, { NameValueTable, NameValueTableRow } from '../../common/SimpleTable';
+import { LightTooltip } from '..';
```
Comment (ashu8912) [inline frontend/src/components/common/Resource/Resource.tsx:38]: looks like this should not belong here

File: frontend/src/components/common/Resource/Resource.tsx [hunk 4]
```diff
@@ -19,16 +19,14 @@ import * as monaco from 'monaco-editor';
...
-import { LightTooltip } from '..';
 import DetailsViewPluginRenderer from '../../../helpers/renderHelpers';
 import { ApiError } from '../../../lib/k8s/apiProxy';
 import {
   KubeCondition,
   KubeContainer,
   KubeContainerStatus,
   KubeObject,
-  KubeObjectInterface
+  KubeObjectInterface,
```
Comment (ashu8912) [inline frontend/src/components/common/Resource/Resource.tsx:29]: this should not belong here

**Review by ashu8912 — COMMENTED** [review]
Tested and it LGTM, just a few comments

File: frontend/src/components/common/Resource/Resource.tsx (hunk 1 above)
Comment (joaquimrocha) [reply frontend/src/components/common/Resource/Resource.tsx:448]: Not sure. It does have a trailing whitespace that should not be there, but it's not part of this patch. Thanks.

File: frontend/src/components/common/Resource/Resource.tsx (hunk 4 above)
Comment (joaquimrocha) [reply frontend/src/components/common/Resource/Resource.tsx:29]: True. I think I copied & paste the entire file due to changing a few things when arranging the commits, and may have removed more than I thought.
